schema/07-coingecko.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE token_values(
+    denom TEXT,
+    current_price NUMERIC,
```

Comment:
I think we can rename this column to be just `price` since it already has a `height` associated to it